### ISSUE TITLE
Stopped working 

### ISSUE THREAD

**Issue by Swiftgti** [acheong08/Bard#76]
Any response from bard, now crashes:

site-packages\Bard.py", line 275, in ask
    chat_data = json.loads(resp.content.splitlines()[3])[0][2]
                           ~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range

**Comment by Swiftgti** [comment]
The issue was because changed my __Secure-1PSID, and the script dont report the Authentication error.

BUT now, apparently the __Secure-1PSID are changing very fast, from one day to another... how to prevent it? any way to automatically retrieve __Secure-1PSID ?

**Comment by raymerjacque** [comment]
nope, its a secure cookie... you have to use the cookie extension and manually copy it every time it changes.

**Comment by acheong08** [comment]
Strange. Mine hasn't changed at all over the past week.

**Comment by hung-scopely** [comment]
@acheong08 maybe you are using a separated account just for Bard and that account isn't used frequently.
